src/vm/ast/definition_sorter/mod.rs

```diff
@@ -254,6 +254,14 @@ impl<'a> DefinitionSorter {
                                     }
                                     return Ok(());
                                 }
+                                NativeFunctions::TupleSet => {
```

Comment:
Does this need to be handled with a special case in this match? Or could TupleSet just be handled like any other native function (i.e., fall through to line 270, which just iterates over the expressions)?

Reply:
#2123

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2117/commits/a1a30823ad531d20310d347a63f1c710ce190742